go/cmd/vtctldclient/command/vreplication/migrate/migrate.go

```diff
@@ -0,0 +1,148 @@
+		StopAfterCopy:             createOptions.StopAfterCopy,
+		NoRoutingRules:            createOptions.NoRoutingRules,
+	}
+
+	_, err := common.GetClient().MigrateCreate(common.GetCommandCtx(), req)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func addCreateFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVar(&createOptions.SourceKeyspace, "source-keyspace", "", "Keyspace where the tables are being moved from.")
+	cmd.MarkPersistentFlagRequired("source-keyspace")
+	cmd.Flags().StringVar(&createOptions.SourceTimeZone, "source-time-zone", "", "Specifying this causes any DATETIME fields to be converted from the given time zone into UTC.")
+	cmd.Flags().BoolVar(&createOptions.AllTables, "all-tables", false, "Copy all tables from the source.")
+	cmd.Flags().StringSliceVar(&createOptions.IncludeTables, "tables", nil, "Source tables to copy.")
+	cmd.Flags().StringSliceVar(&createOptions.ExcludeTables, "exclude-tables", nil, "Source tables to exclude from copying.")
+	cmd.Flags().BoolVar(&createOptions.NoRoutingRules, "no-routing-rules", false, "(Advanced) Do not create routing rules while creating the workflow. See the reference documentation for limitations if you use this flag.")
+
+	cmd.Flags().StringVar(&createOptions.MountName, "mount-name", "", "Name external cluster is mounted as.")
+	cmd.Flags().StringSliceVarP(&createOptions.Cells, "cells", "c", nil, "Cells and/or CellAliases to copy table data from.")
+	cmd.Flags().Var((*topoproto.TabletTypeListFlag)(&createOptions.TabletTypes), "tablet-types", "Source tablet types to replicate table data from (e.g. PRIMARY,REPLICA,RDONLY).")
+	cmd.Flags().BoolVar(&createOptions.TabletTypesInPreferenceOrder, "tablet-types-in-preference-order", true, "When performing source tablet selection, look for candidates in the type order as they are listed in the tablet-types flag.")
+	onDDLDefault := binlogdatapb.OnDDLAction_IGNORE.String()
+	cmd.Flags().StringVar(&createOptions.OnDDL, "on-ddl", onDDLDefault, "What to do when DDL is encountered in the VReplication stream. Possible values are IGNORE, STOP, EXEC, and EXEC_IGNORE.")
+	cmd.Flags().BoolVar(&createOptions.DeferSecondaryKeys, "defer-secondary-keys", false, "Defer secondary index creation for a table until after it has been copied.")
+	cmd.Flags().BoolVar(&createOptions.AutoStart, "auto-start", true, "Start the MoveTables workflow after creating it.")
+	cmd.Flags().BoolVar(&createOptions.StopAfterCopy, "stop-after-copy", false, "Stop the MoveTables workflow after it's finished copying the existing rows and before it starts replicating changes.")
+}
+
+func registerMigrateCommands(root *cobra.Command) {
+	common.AddCommonFlags(migrate)
+	root.AddCommand(migrate)
+	addCreateFlags(createCommand)
+	migrate.AddCommand(createCommand)
+	opts := &common.SubCommandsOpts{
+		SubCommand: "Migrate",
+		Workflow:   "import",
+	}
+	migrate.AddCommand(common.GetCompleteCommand(opts))
+	migrate.AddCommand(common.GetCancelCommand(opts))
```

Comment:
Don't we want show as well?

Reply:
Don't we also want SwitchTraffic?

Reply:
Migrate doesn't have SwitchTraffic, only Complete. https://vitess.io/docs/15.0/reference/vreplication/migrate/